src/Hit/Git/Status.hs

```diff
@@ -27,19 +27,22 @@ data PatchType
     | Unknown
     | BrokenPairing
 
--- | Map conventional characters to 'PatchType'
+-- | Map conventional characters to 'PatchType'.
+-- Renames and copies are respectively C% and D% where % denotes a similarity percentage.
```

Comment:
Let's write this documentation in `{- |` and `-}` style as any other multiline comment. Also, should it be `R%` and `C%`?